molecule/default/playbook.yml

```diff
@@ -8,7 +8,14 @@
       when: ansible_os_family == 'Debian'
 
   vars:
-    cloud_platform: docker
+    - cloud_platform: docker
+    - ansible_python_interpreter: /usr/bin/python3
 
   roles:
     - role: marvel-nccr.aiida
+      vars:
+        aiida_components:
+          - computers
+          - plugins
```

Comment:
just remove this - this is likely the cause for the idempotency failure

Reply:
unfortunately, this means we won't be testing the installation of the plugins + code setup on CI...
but I see no way around it if we want idempotency

Reply:
done